Day-I-Social-Card/social-card/src/App.css

```diff
@@ -2,3 +2,8 @@
   Write your styles/CSS for App.js here and import into the component that you need 
   IF you'd like to attempt to use less you'll have to research how to do this with Create React App
 */
+
+.container{
+  width : 500px;
```

Comment:
Dont want to leave our CSS skills behind. Change this to max-width to make it responsive. you would still need to constrain the card img so it wont create a horizontal scroll bar.